modules/generic/proto.lua

```diff
@@ -32,7 +32,7 @@ local GenericExperts     = require "generic.experts"
 local GenericProto = {}
 
 
-GenericProto.proto = Proto.new("Protobuf", "Google Protobuf Format")
+GenericProto.proto = Proto.new("Protobuf3", "Google Protobuf Format")
```

Comment:
Isn't this supposed to still be for proto2?

Reply:
I am using both Repos so I need to distinguish between their name...